src/Tracing/Tracing.php

```diff
@@ -17,10 +17,17 @@ class Tracing
     /**
      * The point in time where the request was initially started.
      *
-     * @var \Carbon\Carbon
+     * @var float
      */
     protected $requestStart;
 
+    /**
+     * The point in time where the request was initially started.
+     *
+     * @var float|int
+     */
+    protected $requestStartPrecise;
```

Comment:
Definitely don't want to have that twice.

Does Carbon 1 not store microsecond time?

Reply:
See comment explaining what is happening and why.